docs/install/index.md

```diff
@@ -987,10 +987,23 @@ $ make -j $(nproc) USE_OPENCV=1 USE_BLAS=openblas
 
 **Build and install the MXNet R binding**
 
+You will need to first install R. MXNet requires R version >3.3 and we can fetch that from CRAN package repository.
```

Comment:
Please add similar instructions for windows and os x users.

Reply:
for macOS and windows we do not build from source, this will not be there. There is only the issue of ensuring that the pre-built binary in the S3 works fine with both R 3.4 and 3.5. 
And that is not part of this PR.